modules/ROOT/pages/external-libs.adoc

```diff
@@ -49,10 +49,22 @@ of the library's file and validate that the library specified by your user is th
 * `NATIVE`: Native library dependency
 * `JAR`: JAR file dependency
 * `DEPENDENCY`: External dependency (for example, an obtained dependency that is specified in the `pom.xml` if you are using Maven)
+
+[NOTE]
+=======
+For custom connectors, all Maven dependencies must be accessible from Anypoint Exchange. Maven dependencies published to private or public repositories are unaccessible from Anypoint Exchange.
```

Comment:
There is a bit of debate over "unaccessible" and "inaccessible" (with the latter being preferred). I have this solution:
```suggestion
For custom connectors, all Maven dependencies must be accessible from Anypoint Exchange. Maven dependencies published to private or public repositories are not accessible from Anypoint Exchange.
```
lol: when in doubt, reword for simplicity!